test/support/serialization_testing.rb

```diff
@@ -10,4 +17,13 @@ def with_adapter(adapter)
   ensure
     ActiveModel::Serializer.config.adapter = old_adapter
   end
+  alias_method :with_configured_adapter, :with_adapter
```

Comment:
I don't understand the `with_configured_adapter`. Could you show me a situation where it makes sense?